service/ziti-tunnel/cli/cmdlineConstants.go

```diff
@@ -0,0 +1,18 @@
+package cli
+
+import (
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/dto"
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/util/logging"
+)
+
+var GET_STATUS = dto.CommandMsg{
+	Function: "Status",
+}
+
+var templateIdentity = "{{printf \"%40s\" \"Name\"}} | {{printf \"%41s\" \"FingerPrint\"}} | {{printf \"%6s\" \"Active\"}} | {{printf \"%30s\" \"Config\"}} | {{\"Status\"}}\n" +

```

Comment:
would you be able to use the literal string here and make the escaped quotes easier to read?